dashboard/lib/pd/foorm/foorm_parser.rb

```diff
@@ -41,7 +41,9 @@ def self.parse_forms(forms)
       parsed_forms = {general: {}, facilitator: {}}
       forms.each do |form|
         parsed_form = {general: {}, facilitator: {}}
-        form_questions = JSON.parse(form.questions, symbolize_names: true)
+        form_questions = JSON.parse(form.questions)
+        form_questions = ::Foorm::Form.fill_in_library_items(form_questions)
```

Comment:
What's this change?

Reply:
this was to allow us to see results from forms with library items--we weren't filling in library items in the parser before so any library questions would not have been able to be viewed (which is many questions in the new surveys).